src/targets/gpu/CMakeLists.txt

```diff
@@ -235,6 +235,16 @@ else()
     message(STATUS "MIOpen does not have find mode api")
 endif()
 
+check_library_exists(MIOpen "rocblas_gemm_strided_batched_ex" "${MIOPEN_LOCATION}" HAS_ROCBLAS_BETA_FEATURES_API)
```

Comment:
This should check for existence of BETA API e.g. `rocblas_gemm_get_solution()`.

And replace MIOpen with rocBLAS wherever relevant.

Reply:
This has been fixed and now appears to get it right.